eap-xp4-jdk17/image.yaml

```diff
@@ -22,8 +22,6 @@ labels:
 envs:
     - name: DISABLE_LEGACY_SECURITY
       value: "true"
-    - name: AB_JOLOKIA_OFF
```

Comment:
The runtime image needs also to be updated: https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap74-dev/eap-xp4-jdk17/runtime-image/image.yaml#L39